testing/helpers/checkStyleHelper.js

```diff
@@ -0,0 +1,53 @@
+import devices from "core/devices";
+import browser from "core/utils/browser";
+
+function testInChromeOnDesktop(name, testCallback) {
+    if(devices.real().deviceType === "desktop" && browser.webkit) {
+        QUnit.testInActiveWindow.call(null, name, testCallback);
+    } else {
+        QUnit.skip.call(null, name, testCallback);
+    }
+}
+
+function checkTextOverflow(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element.parentNode);
+    assert.equal(style.textOverflow, expectedValue, "textOverflow");
+}
+
+function checkWhiteSpace(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element);
+    assert.equal(style.whiteSpace, expectedValue, "whiteSpace");
+}
+
+function checkOverflowX(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element.parentNode);
+    assert.equal(style.overflowX, expectedValue, "overflowX");
+}
+
+function checkBackgroundColor(assert, element, expectedValue) {
+    let elementBackgroundColor = window.getComputedStyle(element).backgroundColor;
+    let currentElement = element.parentNode;
+    for(let i = 0; i < 5; i++) {
+        let currentStyle = window.getComputedStyle(currentElement);
+        if(elementBackgroundColor !== currentStyle.backgroundColor) {
+            elementBackgroundColor = currentStyle.backgroundColor;
+            break;
+        }
+        currentElement = currentElement.parentNode;
+    }
+    assert.equal(elementBackgroundColor, expectedValue, "backgroundColor");
+}
+
+function checkColor(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element);
+    assert.equal(style.color, expectedValue, "color");
+}
+
+export default {
```

Comment:
Can we export each function separately?

Reply:
Yes, we can export each function separately, without 'export default {':
- import * as CheckStyleHelper from "../../helpers/checkStyleHelper.js";
- CheckStyleHelper = require("../../helpers/checkStyleHelper.js");

But currently all the functions are used in each test.
Are you sure it is worth it at this stage with the only 46 lines of code?

Reply:
Done